src/SapiStatsDispatcher.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Drupal\sapi;
+
+use Drupal\Core\Config\ConfigManager;
+use Drupal\Core\Session\AccountProxy;
+use Drupal\Core\Routing\CurrentRouteMatch;
+use \Drupal\sapi\StatisticsItemInterface;
+
+/**
+ * Class SapiStatsDispatcher.
+ *
+ * @package Drupal\sapi
+ */
+class SapiStatsDispatcher implements SapiStatsDispatcherInterface {
```

Comment:
`Dispatcher` alone would make sense because the class is namespaced. Easier on the eyes.